include/kimera-vio/utils/ColorUtils.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+#include <opencv2/core/core.hpp>
+
+namespace VIO {
+
+/* Color namespace is used to conveniently produce cv color values. The actual
+   Color values are taken from cv::viz:
+   https://github.com/apc-llc/opencv-2.4.10/blob/master/modules/viz/include/opencv2/viz/types.hpp#L191
+
+   cv::viz is a heavy component to include as a mandatory dependency for Color
+   values only, hence the reason for this inclusion.
+*/
+namespace Color {
+  inline cv::Scalar Apricot() { return cv::Scalar(177, 206, 251); }
```

Comment:
Also applying the constexpr suggestion here:
```suggestion
  constexpr cv::Scalar Apricot() { return cv::Scalar(177, 206, 251); }
```

Reply:
`constexpr` requires a literal return type, which `cv::Scalar` is not, so it can't actually be used in this case.